### DISCUSSION TITLE
Storybook-nextjs revalidateTag and fetch nextjs tags

### DISCUSSION THREAD

**Discussion by melrose13-69** [storybookjs/storybook#25834] [Ideas]
### Is your feature request related to a problem? Please describe.

revalidateTag, revalidatePath mock is missing

### Describe the solution you'd like

Ideally, I would like to get the same functionality as in the nextjs application, but you can just mock revalidateTag function

### Are you able to assist to bring the feature to reality?

no

**Comment by valentinpalkovic** [comment]
`revalidateTag` and `revalidatePath` mocks are available in the latest version of Storybook. Please upgrade:

```
npx storybook@latest upgrade
```

If things are still not working, please provide a reproduction.
